chapters/getting_started_linux.md

```diff
@@ -0,0 +1,324 @@
+# Getting started with OpenCL on Ubuntu Linux
+
+OpenCL is included in all the main distributions of Linux. In this guide we choose Ubuntu as one of the most widespread variants, but all of the distributives have some sort of centralized system of package maintaining and OpenCL realizations can be installed from there.
+
+In this guide we're minimally going to use the following tools:
+
+- The command-line
+- C/C++ compiler (gcc & Clang)
```

Comment:
Minor: since we don't say anything about Clang anymore we may want to simplify this:
```suggestion
- C/C++ compiler
```